GetFed/GetFed/Controllers/FoodSearchViewController.swift

```diff
@@ -59,12 +60,36 @@ extension FoodSearchViewController: UITableViewDataSource, UITableViewDelegate {
         guard let cell = tableView.dequeueReusableCell(withIdentifier: Identity.foodSearchResultCell.cellID, for: indexPath) as? FoodResultTableViewCell else {
             fatalError("Fatal error: No cell")
         }
-        cell.foodLabel.text = searchResults?.results[indexPath.row].food.label
-        cell.brandLabel.text = searchResults?.results[indexPath.row].food.brand
+        if let searchResults = searchResults {
```

Comment:
Again, you may want to consider what this will look like if any of the data is missing.

Reply:
Agreed! I've added more robust handling of missing data in my current pull request (commit 
23938e8).